notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -5496,6 +5496,45 @@ mother and daughter [Isabel interacted with her mother, Jennifer.]
 father and daughter [It was strongly suggested that Isabel's father brought her down to his private study late at night to molest her.]
 
 
+movie: The Girl from Monday (2005)
+==================================
+
+:: Title
+The Girl from Monday
+
+:: Description
+The Girl from Monday is a 2005 American film directed by Hal Hartley.
+
+Synopsis: Set in a near-future where virtually everything is commercialized,
+an advertising executive becomes involved with a beautiful woman from another
+planet.
+
+:: Date
+2005
+
+:: References
+https://en.wikipedia.org/wiki/The_Girl_from_Monday
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+extraterrestrial being [The titular girl was a being from a planet in the 147X star system who came to Earth in the guise of a beautiful woman. It was revealed in the end that Jack was really an alien.]
+alien living secretly among us [Two aliens, Jack and the titular girl, were living secretly on Earth in the guise of humans.]
+mass consumerism [The story is a satire on early 2000s consumer society.]
+consumerist dystopia [The story is set in a hyper consumerist society where virtually everything is commercialized, including sexual encounters.]
+resistance movement [The young counter-revolutionary William was leading a movement to return society to its pre-free-market-revolution state.]
+
+:: Minor Themes
+suicide [Jack pointed a gun to his temple in one scene, but didn't go through with pulling the trigger.]
+office romance [Jack refrained from hooking up with his co-worker, Cecile, at the last moment.]
+terrorism [The anti-consumerist counter-revolutionaries were labeled as terrorists by the state.]
```

Comment:
The mind reels. "Counter-revolutionary" implies a revolution. "Anti-consumerist" suggest said Revolution was of a "consumerist" nature? Someone overthrew society in order to make people shop more???

Reply:
That's exactly what happened. I likewise puzzled over that one for awhile, even going to far as to check that I correctly understood the definition of "counter-revolutionary", which, of course, I did.